source/prerequisites/sql-server.txt

```diff
@@ -10,4 +10,117 @@ SQL Server Migration Prerequisites
    :depth: 1
    :class: singlecol
 
-SQL Server.
\ No newline at end of file
+Connecting Relational Migrator to SQL Server requires Change Data 
```

Comment:
This is a bit misleading. The prereqs aren't for connecting to SQL Server, but for running migration jobs. You don't need to do this for creating a project or for converting queries, for example.